tests/core_test/cmaes_test.py

```diff
@@ -0,0 +1,59 @@
+"""Tests CMA ES implementation"""
+
+import jax
+import jax.numpy as jnp
+import pytest
+
+from qdax.core.cmaes import CMAES
+
+
+def test_cmaes() -> None:
+
+    num_iterations = 10000
+    num_dimensions = 100
+    batch_size = 36
+    num_best = 36
+    sigma_g = 0.5
+    minval = -5.12
+
+    def sphere_scoring(x: jnp.ndarray) -> jnp.ndarray:
+        return -jnp.sum((x + minval * 0.4) * (x + minval * 0.4), axis=-1)
```

Comment:
Same thing as above; can't we simply use the standard function defined in tasks, instead of redefining it? (if we notice a bug in the future, it might take more time to fix it in replicated codes.)